dspace-spring-rest/src/test/resources/org/dspace/app/rest/test/metadata-patch-suite.json

```diff
@@ -23,7 +23,7 @@
       ],
       "expect": {
         "dc.title": [
-          { "value": "title 1", "language": null, "authority": null, "confidence": -1 }
+          { "value": "title 1", "language": null, "authority": null, "confidence": -1, "place": 0}
```

Comment:
place should be removed as by discussion in the REST contract PR.
Anyway, if reintroduced it should be tested that its value is respected also when it differs from the order defined by the position when there are multiple values

Reply:
Resolved in the contract PR